playbackSpeed/src/main/kotlin/voice/playbackSpeed/RouletteUtils.kt

```diff
@@ -0,0 +1,30 @@
+package voice.playbackSpeed
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import androidx.core.graphics.ColorUtils
+import kotlin.math.pow
+import kotlin.math.roundToInt
+
+internal object Utils {
```

Comment:
Imo, Utils doesn't make much sense in the kotlin world. I'd replace this by top level functions with a more meaningful name.